_data/collection-index.yml

```
@@ -893,3 +893,8 @@
   contact: https://github.com/git-saj/devcontainer-features/issues
   repository: https://github.com/git-saj/devcontainer-features
   ociReference: ghcr.io/git-saj/devcontainer-features
+- name: igwejk's Java dev container template
+  maintainer: igwejk
+  contact: https://github.com/demos-by-igwejk/devcontainer-templates/issues
+  repository: https://github.com/demos-by-igwejk/devcontainer-templates
+  ociReference: ghcr.io/demos-by-igwejk/devcontainer-templates/java
```

```suggestion
  ociReference: ghcr.io/demos-by-igwejk/devcontainer-templates
```

Hi 👋 

The `ociReference` here refers to the collection metadata artifact instead of the individual 
templates. Hence, the author needs to only add a single entry irrespective of the number of Templates that might exist in their repository.

Can you help update this? thanks!